libr/core/cbin.c

```diff
@@ -1024,12 +1024,18 @@ static int bin_pdb(RCore *core, int mode) {
 }
 
 static int srclineCmp(const void *a, const void *b) {
+	return r_str_cmp (a, b, -1);
+#if 0
```

Comment:
Why not just kill it?

Reply:
yes let's remove this please.
`r_str_cmp` seems exactly what we need!